### PR TITLE
Fixes AASX upload/preconfig ignoring thumbnail

### PR DESCRIPTION
**Pull request by mdanish98**
Signed-off-by: Mohammad Ghazanfar Ali Danish <ghazanfar.danish@iese.fraunhofer.de>

### CODE REVIEW

File: basyx.aasrepository/basyx.aasrepository-backend/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/backend/CrudAasRepository.java [hunk 1]
```diff
@@ -44,6 +44,7 @@
 import org.eclipse.digitaltwin.basyx.core.pagination.CursorResult;
 import org.eclipse.digitaltwin.basyx.core.pagination.PaginationInfo;
 import org.eclipse.digitaltwin.basyx.core.pagination.PaginationSupport;
+import org.eclipse.digitaltwin.basyx.http.Base64UrlEncodedIdentifier;
```
Comment (FrankSchnicke) [inline basyx.aasrepository/basyx.aasrepository-backend/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/backend/CrudAasRepository.java:47]: Please revert not needed changes.

File: basyx.aasrepository/basyx.aasrepository-backend/pom.xml [hunk 2]
```diff
@@ -27,5 +27,9 @@
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
 		</dependency>
+		<dependency>
```
Comment (FrankSchnicke) [inline basyx.aasrepository/basyx.aasrepository-backend/pom.xml:30]: Why is this change needed?

**Review by FrankSchnicke — CHANGES_REQUESTED** [review]

File: basyx.aasrepository/basyx.aasrepository-backend/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/backend/CrudAasRepository.java (hunk 1 above)
Comment (mdanish98) [reply basyx.aasrepository/basyx.aasrepository-backend/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/backend/CrudAasRepository.java]: Reverted this

File: basyx.aasrepository/basyx.aasrepository-backend/pom.xml (hunk 2 above)
Comment (mdanish98) [reply basyx.aasrepository/basyx.aasrepository-backend/pom.xml]: It was unnecessary, hence removed.
